modules/database/funcs.go

```diff
@@ -229,3 +232,36 @@ func (d *Database) getDBWithName(
 
 	return nil
 }
+
+//
+// Returns a *Database object with specified name and namespace
```

Comment:
comment needs to start with GetDatabaseByName

Reply:
done